autrainer/core/scripts/preprocess_script.py

```diff
@@ -133,19 +132,25 @@ def _preprocess_datasets(self) -> None:
                 instance_of=AbstractFileHandler,
             )
             output_file_type = dataset["file_type"]
-            
+
             # override dataset file handling to work with raw audio
             dataset["file_handler"] = preprocess["file_handler"]
             # None allows dataset to work with all audio files
             dataset["file_type"] = None
             dataset["seed"] = 0  # ignored
             dataset["batch_size"] = 8  # ignored
-            data = autrainer.instantiate_shorthand(dataset)
+            _ = dataset.pop("criterion")
+            _ = dataset.pop("transform")
+            features_path = dataset.pop("features_path")
```

Comment:
I think this would be more clear if we remove the if statement below and do `features_path = dataset.pop("features_path", dataset["path"])` instead? If you find it more complicated just keep it as is.

Reply:
This will fail if "features_path" is specified but is `None`. So I'll stick to my version